src/cogs/cmds/econ.py

```diff
@@ -1044,7 +1044,7 @@ async def mine(self, ctx):
 
             # only works cause num of pickaxes is 6 and levels of fake finds is 3
             # please don't bug me about jank code, I know
-            fake_finds = self.d.mining.finds[math.floor(self.d.mining.pickaxes.index(pickaxe) / 2)]
+            fake_finds = min(self.d.mining.finds[math.floor(self.d.mining.pickaxes.index(pickaxe) / 2)],3)
```

Comment:
What is this supposed to do?

Reply:
so that the fake finds aren't broken by pickaxe number 7 in a function built for 6 pickaxes

Reply:
That will break, finds[3] will be out of bound, that should be min(<some stuff>, 2) if you want that to work

Reply:
oh thanks. I must have some math wrong